docs/site/translation.md

```diff
@@ -86,17 +86,23 @@ Coverage for other languages is at comprehensive.
 <!-- I don't think we want to encourage this: ... if there is a need to have coverage at lower level in some locale,
 please file a ticket. [CLDR-18283](https://unicode-org.atlassian.net/browse/CLDR-18283) -->
 
-#### ISO 8601 calendar
+#### Gregorian Calendar (Year First) calendar
 
 This is a variant of the Gregorian calendar whose formats always use year-month-day ordering and a 24-hour time cycle.
-The English name has changed to reflect that (and also added a variant); locales should update accordingly:
-- calendar-iso8601: Gregorian (Year First)
-- calendar-iso8601-variant: ISO 8601 Order
+*Note: the code is `iso8601`, but disregard that; it will be changed after submission.*
 
-Please go though the ISO8601 fields. You should change separators to match what is acceptable in your language. However, do not change the ordering of the elements, which should be strictly the following order (for any that occur in a particular pattern):
-* era - year - month - day - day-of-week - hour - minute - second
+Page | Code | English
+-|-|-
+Date & Time / Gregorian YMD | yMMMd | y MMM d
+Date & Time / Gregorian YMD | … | …
+Locale Display Names / Keys | calendar-iso8601 | Gregorian Calendar (Year First)
 
-Also avoid changing the width of numeric fields (like `dd`).
+Please go though the Gregorian Calendar (Year First) fields.
+* You should change separators to match what is acceptable in your language. 
+* The ordering of fields in the pattern should be strictly the following order (for any that occur in a particular pattern).
```

Comment:
```suggestion
* The ordering of fields in the pattern should be strictly the following order (for any that occur in a particular pattern):
```

Reply:
Fixed this one (text was different so I couldn't commit the suggestion).